app/sources/session.jsx

```diff
@@ -0,0 +1,26 @@
+import config from 'config';
+
+export default class SessionSource {
+  static urlRoot = `${config.apiPath}/users`
+
+  static headers = {
+    'Accept': 'application/json',
+    'Content-Type': 'application/json'
+  }
```

Comment:
these headers seems to be used really widely, what do you think about extracting them into self defined `fetch` lib, probably we could define `fetch` and `fetchAuth` for requests having user auth token?